CodeStarterCamp_Week2/LottoGenerator.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  LottoNumberChecker.swift
+//  CodeStarterCamp_Week2
+//
+//  Created by JIWOONG on 2023/08/01.
+//
+
+import Foundation
+
+var lottoNumbersDictionary: [String: [Int]] = [:]
+
+func makeLottoNumbers() -> [Int] {
+    var randomNumber: Int
+    var lottoNumbers: Set<Int> = []
+    var sortedLottoNumbers: [Int] = []
+    while lottoNumbers.count < 6 {
+        randomNumber = Int.random(in: 1...45)
+        lottoNumbers.insert(randomNumber)
+    }
+    sortedLottoNumbers = Array(lottoNumbers).sorted()
```

Comment:
Set 타입의 인스턴스 메서드 `sorted()`는 반환 타입이 Array이기 때문에
별도로 변수를 만들어주지 않고 바로 반환해주셔도 괜찮습니다!
```suggestion
    return lottoNumbers.sorted()
```